README.md

```diff
@@ -1 +1,169 @@
-# Metrics-App
\ No newline at end of file
+# Metrics App
+
+<a name="readme-top"></a>
+
+
+<a name="readme-top"></a>
+
+
+  <h3><b>Metrics App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Metrics App](#metrics-app)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Metrics App\] ](#-metrics-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/capstone/react_capstone.md?plain=1#L87), kindly deploy the app and include the link within your readme file.